src/swish/swish-test.ms

```diff
@@ -228,8 +240,9 @@
   )
 
 (isolate-mat select-tests ()
+  (define subdir (uuid->string (osi_make_uuid)))
   (define test-file
-    (write-test-file "test-specified.ms"
+    (write-test-file (path-combine "test-specified.ms")
```

Comment:
Are you missing the path argument to path-combine?

Reply:
Well spotted. I meant to use the subdir to check the case where we `--rerun` all results in a particular directory.

Pushed a fix for that along with some proposed breaking changes.